src/statement/dataTypes.ts

```diff
@@ -88,6 +91,10 @@ export const getFireboltType = (type: string): string => {
   return mappedType || key;
 };
 
+export const isByteAType = (type: string) => {
+  return "bytea" === type;
```

Comment:
what if `bytea`  will be nullable. 
I propose to use something like 

```typescript
export const BYTEA_TYPES = withNullableTypes(["bytea"]);
export const isByteAType = (type: string) => {
  return BYTEA_TYPES.indexOf(type) !== -1
}
```

Reply:
I actually tried this scenario (I have a test with a null bytea), and the result seems alright:

```
    const statement = await connection.execute("SELECT null::bytea");

    const { data, meta } = await statement.fetchResult();
    expect(meta[0].type).toEqual("nullable(bytea)");
    const row = data[0];
    expect((row as unknown[])[0]).toEqual(null);
```